contracts/Scroll_SpokePool.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
```

Comment:
Pretty sure we've bumped up to ^0.8.22 or 0.8.23 elsewhere - might be worth following that here.

Reply:
Not sure we want to do this. The `^0.8.0` denotes we want the latest version within `^0.8.X`

Reply:
yeah if hardhat.config specifies 0.8.23 then this will be compiled with that